bin/init-cluster

```diff
@@ -30,7 +30,7 @@ options = {
   username: ENV.fetch("CB_USERNAME", "Administrator"),
   password: ENV.fetch("CB_PASSWORD", "password"),
   bucket: ENV.fetch("CB_BUCKET", "default"),
-  server_quota: ENV.fetch("CB_SERVER_QUOTA", 512).to_i,
+  server_quota: ENV.fetch("CB_SERVER_QUOTA", 1024).to_i,
```

Comment:
This doesn't seem related to thread_sanitizer

Reply:
I will eject this into separate PR